src/index.js

```diff
@@ -49,13 +50,13 @@ const init = async () => {
     }));
   } else {
     app.use(helmet({
-      noCache: true,
+     // noCache: true,
       frameguard: {
         action: 'deny',
       },
     }));
   }
-
+  app.use(noCache());
```

Comment:
was this change made due to some error? does helmet remove support of noCache in json object?

Reply:
yeah its been removed with a solution to add nocache separately it was causing loading issues so when I removed the offending code it removed the message and ran slightly quicker, we also have a squalize deprecation warning but as that didn't effect normal operations I left it in there I am hoping that the node upgrade will resolve that one

Reply:
sequilize warning is resolved https://dfe-secureaccess.atlassian.net/browse/DSI-5944 @Avinash-teky but has not been merged yet